### PR TITLE
New subpackage foreman-proxy-journald

### PR DESCRIPTION
**Pull request by lzap**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly

### CODE REVIEW

**Comment by lzap** [comment]
All green as well.

**Comment by lzap** [comment]
Subpackage installer PR: https://github.com/theforeman/puppet-foreman_proxy/pull/461

**Comment by ekohl** [comment]
For the installer I'd like parity between debs and rpms here. @mmoll can we also introduce one for Debian as well?

**Comment by lzap** [comment]
Can we have this merged in order to unblock nightlies, we can catchup tomorrow either @mmoll does it or I will do changes in the installer for Debian.

**Comment by mmoll** [comment]
GH-3098

**Comment by lzap** [comment]
Rebased.

**Comment by ekohl** [comment]
[test rpm]

File: packages/foreman/foreman-proxy/foreman-proxy.spec [hunk 1]
```diff
@@ -195,6 +213,9 @@ fi
 
 
 %changelog
+* Tue Oct 16 2018 Lukas Zapletal <lzap+rpm@redhat.com> - 1.21.0-0.2.develop
```
Comment (ekohl) [inline packages/foreman/foreman-proxy/foreman-proxy.spec]: Could you update the date?

Comment (lzap) [reply packages/foreman/foreman-proxy/foreman-proxy.spec]: Sure done.

**Comment by lzap** [comment]
Ok, what is the next step for me? I would love to have this merged ASAP.

**Comment by tbrisker** [comment]
should this also be cp-ed to 1.20-stable branch?  https://community.theforeman.org/t/foreman-1-20-test-week/11618/20

**Comment by ekohl** [comment]
You're right.

**Pull request merged**